CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 2023-07-26 v0.19.0
+## 2023-08-16 v0.19.0
+
+> Note that the `--topic` CLI option is being depreaced in favour of two more spefici options `--pubsub-topic` and `--content-topic`.
```

Comment:
```suggestion
> Note that the `--topic` CLI option is being deprecated in favor a more specific option `--pubsub-topic`.
```

I don't foresee that `content-topic` will replace `pubsub-topic` - it will be an alternative, sure, when autosharding is in place, but I don't think we have the context to mention this here.

Reply:
Also, as much as it pains me, we try to use US English for publications, so "favor" instead of "favour" 😢